app/src/main/java/com/mapbox/maps/testapp/examples/GlobeActivity.kt

```diff
@@ -39,13 +40,20 @@ class GlobeActivity : AppCompatActivity() {
   private lateinit var mapboxMap: MapboxMap
   private lateinit var infoTextView: TextView
 
+  private fun ProjectionName.toMapProjection() = when (this) {
+    ProjectionName.MERCATOR -> MapProjection.Mercator
+    ProjectionName.GLOBE -> MapProjection.Globe
+  }
```

Comment:
nit: we can remove the conversion and use `MapProjection` directly instead. This is a bit confusing in this activity, as it only uses `MapboxMap#setMapProjection(ProjectionName)` API.

Reply:
it uses runtime styling updates below in this activity

Reply:
sorry, just noticed we are using style.setProjection API as well below, I think we could be consistent and use `MapboxMap#setMapProjection(ProjectionName)` API in both places and avoid the conversion.

Reply:
I've added this function instead of hardcoding to check that setting initial value as mercator works as expected as well